common/pulp/common/constants.py

```diff
@@ -13,3 +13,9 @@
 TASK_FINAL_STATES = (TASK_STATES.SKIPPED, TASK_STATES.COMPLETED, TASK_STATES.FAILED,
                      TASK_STATES.CANCELED)
 TASK_INCOMPLETE_STATES = (TASK_STATES.WAITING, TASK_STATES.RUNNING)
+
+# The interval in seconds for which the Celery Process monitor thread sleeps between
+# checking for missing Celery processes.
+CELERY_CHECK_INTERVAL = 60
```

Comment:
I do think they should be moved out of pulp/common/constants though. That area is more formally reserved in pulp3 for code that is common to the server and client. These constants will never be accessed on the client side so they can be moved into somewhere in pulp.tasking.